README.md

```diff
@@ -2,6 +2,11 @@
 [![Build Status](https://platform-ci.scala-lang.org/api/badges/scalameta/scalameta/status.svg)](https://platform-ci.scala-lang.org/scalameta/scalameta)
 [![Join the chat at https://gitter.im/scalameta/scalameta](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/scalameta/scalameta?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+## Versions
+Release: ![Release](https://img.shields.io/maven-central/v/org.scalameta/scalameta_2.12.svg)
```

Comment:
How about?

> See http://scalameta.org/#Gettingstarted for the latest stable and pre-release versions.

Those version numbers are copy-pasteable. The pre-release version is updated on every merge into master.